+nix/Group.m

```diff
@@ -17,6 +17,7 @@
     methods
         function obj = Group(h)
             obj@nix.NamedEntity(h);
+            obj@nix.MetadataMixIn();
             
```

Comment:
Ah good catch. SourcesMixIn does not have a specific constructor, therefore it did not show up in the SourcesMixIn methods tests.